static/app/utils/metrics/index.tsx

```diff
@@ -208,9 +208,10 @@ export function getMetricsApiRequestQuery(
     useCase,
     interval,
     groupBy,
+    orderBy,
     allowPrivate: true, // TODO(ddm): reconsider before widening audience
     // max result groups
-    per_page: 10,
+    limit: 10,
```

Comment:
Not sure if this function is relevant, but: For DDM usage, I believe we want no limit (only the one that the backend will enforce for safety reasons). For dashboards, I think there's a user-configurable limit for charts and no limit for tables.

Reply:
related to the other comment per_page parameter is used by the old metrics layer, limit is not used by either new or the old layers